movies/3.api-doc.md

```diff
@@ -0,0 +1,89 @@
+# The API documentation
+
+## Story
+
+As a client developer, I want to find human-readable documenation of the API.
+As a client, I want to find a machine-readable description of the API capabilities.  
+
+## API behaviour
+
+```http
+GET https://hydra-movies.herokuapp.com/
+```
+
+Response headers (excerpt):
+
+```http
+Link: </doc>; rel="http://www.w3.org/ns/hydra/core#apiDocumentation"
+```
+
+```http
+GET https://hydra-movies.herokuapp.com/doc
+```
+
+Response body (excerpt):
+
+```json
+{
+  "@context": "/context.jsonld",
+  "@id": "/doc",
+  "@type": "ApiDocumentation",
+  "title": "Hydra Movies API",
+  "description": "This API shows how a basic CRUD (Create, Read, Update, Delete) API can be designed using Hydra Core Vocabulary.",
+  "entrypoint": "/",
+  "supportedClass": [
+    {
+      "@id": "schema:EntryPoint",
+      "@type": "Class",
+      "title": "API entrypoint",
+      "description": "Every capability of the API can be reached from here by following links."
+    }
+  ]
+}
+```
+
+### Explanation
+
+```http
+Link: </doc>; rel="http://www.w3.org/ns/hydra/core#apiDocumentation"
+```
+
+```http
+GET https://hydra-movies.herokuapp.com/doc
+```
+
+As described in ["API entrypoint"](./1.entry-point.md), every API response has to contain a Link to the documentation. This allows a client to dereference and process the documentation, even if it does not start from the entrypoint (e.g. because of a "deep link" or a bookmark from a former visit).
+
+```
+"@context": "/context.jsonld",
```

Comment:
Maybe remove from here is not really relevant?

Reply:
Are you refering to the whole section or only the `@context` line?

Reply:
Just the context like, I think 🤔